althea_kernel_interface/src/exit_server_counter.rs

```diff
@@ -117,7 +116,7 @@ impl KernelInterface {
                 &format!("tmp_{}", target.set_name()),
                 target.set_name(),
             ],
-        )?;
```

Comment:
What's wrong with using `?`?

Reply:
nothing, I just swaped this with a warn statement.